From ba3c502e1894842dfa6de0077a431e8be375f70b Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Sun, 28 Jul 2019 00:19:51 +0200
Subject: [PATCH 0252/1239] usb: hub: increase usb port reset time as a W/A for
 some usb hubs

Some usb hubs are causing problems during usb hub port reset. Work-around
for this issue is increasing the HUB_SHORT_RESET_TIME and was recommended
by HW team.

Change-Id: Iadb3ba22ebd0bbbb09619b342ef2d1becbcf3ab7
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
---
 common/usb_hub.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/usb_hub.c b/common/usb_hub.c
index 25c2ac4345..0e3d775743 100644
--- a/common/usb_hub.c
+++ b/common/usb_hub.c
@@ -41,7 +41,7 @@
 
 #define USB_BUFSIZ	512
 
-#define HUB_SHORT_RESET_TIME	20
+#define HUB_SHORT_RESET_TIME	100
 #define HUB_LONG_RESET_TIME	200
 
 #define PORT_OVERCURRENT_MAX_SCAN_COUNT		3
-- 
2.29.0

